gtfu/src/main/java/gtfu/GPSStats.java

```diff
@@ -0,0 +1,48 @@
+package gtfu;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+public class GPSStats {
+
+    private static final DecimalFormat df = new DecimalFormat("0.00");
+    public int maxUpdateTime = 0;
+    public int minUpdateTime = 999999;
+    public float avgUpdateTime = 0;
+
+    public GPSStats(Map<String, GPSData> latLonMap) {
```

Comment:
It would be good form to have a comment that says this constructor depends on the lat long map being ordered by time

Reply:
It doesn't, though! DayLogSlicer does rely on the query from DB.java being ordered by timestamp. I mention this in a comment in DayLogSlicer, but I'll add a comment in DB.java as well.